packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -673,6 +688,20 @@ class LoggingController extends DisposableController
                 log.levelName.caseInsensitiveContains(substring);
             if (matchesLevel) return true;
 
+            final matchesIsolateName =
+                log.isolateRef?.name?.caseInsensitiveContains(substring) ??
+                    false;
+            if (matchesIsolateName) return true;
+
+            final zone = log.zone;
+            final matchesZone =
```

Comment:
[suggestion] break this into two variables (e.g. `matchesZoneName` and `matchesZoneIdentity`) to make this more readable